docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+# Simple docker-composer.yml
+# You can change container_name, port or volume location
+
+version: '3.3'
```

Comment:
Maybe change this to the latest `3.8` version [[source](https://docs.docker.com/compose/compose-file/)]